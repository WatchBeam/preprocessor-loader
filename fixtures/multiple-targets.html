<!-- @if PLATFORM == 'xbox' -->
<a></a>
<!-- @endif -->
<!-- @if locale != 'en' -->
<b></b>
<!-- @endif -->
<!-- @if locale == 'en' -->
<c></c>
<!-- @endif -->
<!-- @if PLATFORM != 'desktop' -->
<d></d>
<!-- @endif -->
