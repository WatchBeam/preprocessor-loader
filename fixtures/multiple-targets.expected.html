<a></a>
<c></c>
<d></d>
